i18n/en.i18n.json

```diff
@@ -11,7 +11,7 @@
   "admin": "Admin",
   "post": "Post",
   "toolbox": "Toolbox",
-  "sign_up_sign_in": "Sign Up/Sign In",
+  "sign_up_sign_in": "Register/Sign In",
```

Comment:
Yeah, you're right, it makes sense.
...althought the italian, i.e., is already using register from the beginning (there's no *sign up* equivalent...)